### PR TITLE
Modified CS_UCD to recover from service issues.  Some rewording.

### PR DESCRIPTION
**Pull request by trjaffe**
Issue #27  should be fixed.  It's from the service, so now there's just more error trapping.  A few misc. improvements.

### CODE REVIEW

File: CS_UCDs.ipynb [hunk 1]
```diff
@@ -248,9 +253,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python [conda env:pyvo-dev]",
    "language": "python",
-   "name": "python3"
+   "name": "conda-env-pyvo-dev-py"
```
Comment (stargaser) [inline CS_UCDs.ipynb:258]: The continuous integration check did not like the kernel name.

`[nbcollection (ERROR)]: Notebook 'CS_UCDs.ipynb' errored: No such kernel named conda-env-pyvo-dev-py`

File: CS_UCDs.ipynb [hunk 2]
```diff
@@ -152,14 +149,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "coord = SkyCoord.from_name(\"m83\")\n",
-    "query=f\"select top 10 {ra_name}, {dec_name} from dbo.DetailedCatalog\"\n",
     "#  Look for all TAP services with x-ray and optical data\n",
     "collection={}\n",
     "for s in vo.regsearch(servicetype='tap',keywords=['x-ray','optical']):\n",
-    "    if \"wfau\" in s.ivoid:  continue #  These sometimes have issues\n",
+    "    #if \"wfau\" in s.ivoid:  continue #  These sometimes have issues\n",
```
Comment (stargaser) [inline CS_UCDs.ipynb:155]: The `try-except` below makes this line unnecessary so it might be clearer to delete instead of commenting out.

**Review by stargaser — COMMENTED** [review]
The modified notebook ran for me on my laptop. CircleCI wanted a different kernel name. One other minor comment inline.

**Review by tomdonaldson — APPROVED** [review]
These changes look great.  The only CI problem is with the metadata for `CS_UCDs.ipynb`, which should probably be updated prior to merge.

**Pull request merged**